deno/send_message/README.md

```diff
@@ -0,0 +1,101 @@
+# SendMessage() 
+
+A sample Deno Cloud Function sending a message using a specific channel to a receiver
+
+Supported channels are `SMS`, `EMAIL` ,`DISCORD` and `TWITTER`.
+
+
+_Example function payloads:_
```

Comment:
The example payloads could be cleaned up a bit to make them more readable, each JSON field is usually formatted to be on its own line to make it easier to understand and read.